contribute/documentation/how/how-the-docs-is-built.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
I think this content should be put in different places:

- The docs is about how to contribute. Reference information on how the devdocs is deployed should go to the README in https://github.com/PrestaShop/devdocs-site/
- Information on how to run the site locally should be updated [here](https://devdocs.prestashop.com/1.7/contribute/documentation/how/edit-on-your-computer/) (this is tricky because it involves using submodules - personally I cloning `devdocs-site`, then working on the submodule directory and then push to my `docs` remote, I will probably have to spend some time working out a tutorial)
- Branch information should be added as a "Documentation versions" subsection [here](https://devdocs.prestashop.com/1.7/contribute/documentation/how/). I think it would be best to add it at the top, before "Edit on Github".

Keep in mind that it's likely that whole contributing section will eventually be moved from the devdocs to the project's site, because it's project related, not PrestaShop reference.

Reply:
> The docs is about how to contribute. Reference information on how the devdocs is deployed should go to the README in https://github.com/PrestaShop/devdocs-site/

You can contribute to the theme 😄  and then you need to understand how it's being deployed because you need to submit changes in a relevant way to be added.

But I think I understand what you mean. You mean "the docs is about how to contribute **to content**".

Reply:
@eternoendless 

>  Reference information on how the devdocs is deployed should go to the README in https://github.com/PrestaShop/devdocs-site/

I'm afraid someone will not find it if we put it in https://github.com/PrestaShop/devdocs-site/ . I don't expect a lot of people to go on https://github.com/PrestaShop/devdocs-site/ repository. I will at least add a link.